docs/general/serializers.md

```diff
@@ -74,6 +74,55 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+As of release `0.10`, polymorphic relationships are serialized just by specifying the relationship. For example:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable
+end
+```
+
+If the instance of `Picture` belongs to a `Product`, an example payload might be:
+
+```javascript
+{
+  picture: {
+    id: 1,
+    imageable: {
+      id: 3,
+      title: "Product 3"
+    }
+  }
+}
+```
+
+In prior releases (and optionally in `0.10`), a `polymorphic: true` parameter is required:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable, polymorphic: true
+end
+```
+
+In this case, an example payload might be:
```

Comment:
Pardon the ambiguity. I originally uncovered this behavior with the default adapter, Attributes. I went back and confirmed on 0.9.4 that the option actually is not required there either. I will remove the content about the option.